specs/spec-2-election-ui.md

```diff
@@ -0,0 +1,200 @@
+
+- I want to see a candidate profile (Near Social) and the platform they used to nominate themselves (nomination mechism).
+- I want to see a candidate's past congressional history on profile (past roles, proposals, and complaints), so I can audit their past performance.
+- I want to see the profile + attestation (queried from the nomination process) for a candidate so I can see social proof for why someone was elected.
+- I want to see what DAOs a candidate is in even if they are not part of NDC, in order to understand what communities they are a part of.
+
+#### Runoffs
+
+- I want to know whether the election resulted in any tie, and what are the clear next steps to finalizing runoff.
+
+### Watchdog
+
+- I want to see details about the ongoing election to judge if it's worth my time and effort to join the NDC and be heard. 
+
+### Candidate
+
+- I want elections to be relatively short so that the time I spend campaigning can be focused. 
+- I want to see how much of the voting body has voted in total so I can plan campaign efforts.
+
+### Additional Features & Requirements
+
+<!-- Optionally, describe: Expected functionality, Security issues/requirements, Assumptions/dependencies (internal and external), Potential problems/roadblocks, Budget concerns. -->
+
+- Edit profile in the scope of near social profile
+- Results -> green check mark
+- Votes can not be changed once cast, but can be changed before they are. 
+
+## Tech Spec
+
+<!-- How does the product implement the features and requirements? Not every feature or requirement needs deep consideration here, but implementation considerations are discusssed they should be captured here. -->
+
+### Smart Contract Functions
+
+<!-- What functions and functionalities should the widget have -->
+
+### Data Structures
+
+### UML Diagrams
+
+<!-- Include diagrams that best capture what needs to be built. Sequence, State, Interaction, Activity, Etc.-->
+
+### Technical Dependencies 
+
+Voting Contracts @ https://github.com/near-ndc/voting-v1
+
+## Design Specification
+
+<!-- UX and UI that affords all of the features and requirements. How it behaves and what it looks like. -->
+
+### Wireframes / Screens
+
+- [2023-05-30 v0.1.0](https://www.figma.com/file/OTKmz4ga22khc2vtWPFkzB/Election-UI-Wireframe?type=design&node-id=102%3A2&t=rNzg02hFvCuYLv9a-1)
+  - Desktop Voting for review
+  - Roughed out Desktop Profile & Desktop Admin
+
+### UI Design
+
+Follow the NDC Brand Book in line with Material Design, falling back on vanilla [Bootstrap](https://getbootstrap.com/).
+
+![Primary and Secondary NDC Colors in a Material Design Pallet](https://i.imgur.com/Rzx9XnX.jpg)
+
+### Dependent Widgets
+
+<!-- Does the widget interact with other widgets? -->
+
+- Kudos Widget
+- Near Social Profile
+
+## Open Questions and Comments
+
+<!-- Topics not covered in this document that need to be addressed in a meeting or async. Potential future improvements, Summary of open discussion. -->
+### Open
+
```

Comment:
let's add a note that we will also need an indexer. We should create a separate RFP (based on this spec) to provide indexer features:
* query candidates
* query votes
* query ongoing results.

Suggestion: let's add it in the section above under: Tech Spec / Dependent Backend

Reply:
These have been captured under ## Tech Spec ### Smart Contract Functions. Can we capture enough detail here that the developers working on this spec are responsible for figuring out how to get data from the smart contract? How do most BOS widgets get their information today? Can that same technique be used for Election UI?